web/docs/setup-guides/update-a-node.md

```diff
@@ -0,0 +1,31 @@
+---
+sidebar_position: 2
+---
+
+# Update a node
+
+To update a [deployed node](./deploy-a-node/README.md), proceed with the
+platform-specific instructiosn below.
+
+## Docker
+
+Run the following two commands to update a Docker deployment:
+
+```bash
+docker compose down
+docker compose up --pull
```

Comment:
Should be the following to minimize downtime:

```
docker compose pull
docker compose down
docker compose up --detach
```

For the configurator we create `docker-compose.NODE.yaml` files, so we may actually want to write this instead:

```
docker compose -f docker-compose.NODE.yaml pull
docker compose -f docker-compose.NODE.yaml down
docker compose -f docker-compose.NODE.yaml up --detach
```

Reply:
@dominiklohmann can you take a look at the revised instructions? I integrated your suggestions.